modules/detection/VADReportingService.js

```diff
@@ -0,0 +1,247 @@
+import EventEmitter from 'events';
```

Comment:
Is this class used currently? I can't find a reference for it?

Reply:
Noup, it was part of the preparation to implement 'recommended device notification' but that got deferred.